main/windows.js

```diff
@@ -83,14 +83,30 @@ function hideLauncherWindow() {
 
 function openAppWindow(app) {
     const lastWindowState = settings.loadLastWindow();
+
+    let { x, y } = lastWindowState;
+    const { width, height } = lastWindowState;
+    if (x && y) {
+        const { bounds } = electron.screen.getDisplayMatching(lastWindowState);
+        const left = Math.max(x, bounds.x);
+        const top = Math.max(y, bounds.y);
+        const right = Math.min(x + width, bounds.x + bounds.width);
+        const bottom = Math.min(y + height, bounds.y + bounds.height);
+        if (left > right || top > bottom) {
```

Comment:
Why is the logic with `Math.min/max` needed? Would this be also sufficient?
```js
  const windowsLeftStart = x;
  const screensRightEnd = bounds.x + bounds.width;
  const windowsTopStart = y;
  const screensBottomEnd = bounds.y + bounds.height;
  if (windowsLeftStart > screensRightEnd || windowsTopStart > screensBottomEnd)
```

Reply:
Here we come to a windows issue again. A fullscreen window reports both topleft and bottomright corners outside of the display bounds :\
So this solution is looking for the intersection of the closest matching display against the window bounds.